libs/fluxc-plugin/src/main/kotlin/org/wordpress/android/fluxc/network/rest/wpcom/wc/bookings/BookingsRestClient.kt

```diff
@@ -19,6 +20,25 @@ class BookingsRestClient @Inject constructor(
         const val DEFAULT_PER_PAGE = 25 // Number of items to fetch in a single request
     }
 
+    suspend fun fetchBooking(
+        site: SiteModel,
+        bookingId: Long
+    ): WooPayload<BookingDto> {
+        val endpoint = WCWPAPIEndpoint(
+            WOOCOMMERCE.bookings.id(bookingId).endpoint
+        ).pathV2Bookings
```

Comment:
I missed this in my review.

```suggestion
        val endpoint = WOOCOMMERCE.bookings.id(bookingId).pathV2Bookings
```

Reply:
Done in https://github.com/woocommerce/woocommerce-android/pull/14750/commits/5d68d5053d7e953a8c788c6089dc30d31d0895a8